src/app/index.html

```diff
@@ -0,0 +1,95 @@
+<!doctype html>
```

Comment:
Absolutely. Gulp's collectstatic call (by way of django) only pulls css and js files.